the-collab-lab/tcl-26-smart-shopping-list
# Add number of items in list

## Description



I've added an area to the Shopping List Header.js that displays the number of items in your shopping list.

For now, I do not have the layout finalized. And I could use feedback on how to get the text to display differently mobile.

## Related Issue



Closes #83 : Add item count to header

## Acceptance Criteria



## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before



![image](https://user-images.githubusercontent.com/61371242/131434997-5c6d1b6d-9981-42eb-bae5-3eb91ad17e5c.png)


### After



![image](https://user-images.githubusercontent.com/61371242/131435017-d8444ef9-d66a-4be8-9963-e3c4f0075a41.png)


## Testing Steps / QA Criteria



1. Pull branch
2. Make sure the number of items displayed matches the number of lists in item.
